Rock.Rest/Controllers/TagsController.Partial.cs

```diff
@@ -147,12 +148,13 @@ public IQueryable<Tag> AvailableNames( int entityTypeId, int ownerId, Guid entit
         /// <param name="entityGuid">The entity unique identifier.</param>
         /// <param name="name">The name.</param>
         /// <param name="entityQualifier">The entity qualifier.</param>
+        /// <param name="categoryIds">The delimited list of category id </param>
         /// <returns></returns>
         [Authenticate, Secured]
         [HttpGet]
-        public IQueryable<Tag> AvailableNames( int entityTypeId, int ownerId, Guid entityGuid, string name, string entityQualifier )
+        public IQueryable<Tag> AvailableNames( int entityTypeId, int ownerId, Guid entityGuid, string name, string entityQualifier, string categoryIds = null )
         {
-            return AvailableNames( entityTypeId, ownerId, entityGuid, name, entityQualifier, string.Empty );
+            return AvailableNames( entityTypeId, ownerId, entityGuid, name, entityQualifier, string.Empty, categoryIds );
```

Comment:
CategoryIds which can optionally be passed through  querystring is mentioned in all AvailableNames method.